src/pages/ProAmmPool/PositionListItem.tsx

```diff
@@ -34,6 +34,25 @@ import { unwrappedToken } from 'utils/wrappedCurrency'
 
 import ContentLoader from './ContentLoader'
 
+const DisabledButtonOutlined = styled(ButtonOutlined)`
```

Comment:
@namgold found the reason of disable button with mouse over tooltip. Just change props name from `disabled` to `isDisabled` or something else, it'll work

Reply:
or Diep's style: `$disabled`

Reply:
I dont like $ sign on variable name. It likes JQuery...

Reply:
it's like PHP :))

Reply:
Nice. Just found out that SC supports transient props. https://styled-components.com/docs/api#transient-props

Reply:
its a feature of styled-component not my style :))). It look unfamiliar at first point but later we will know it only use for SC styling

Reply:
for example: if we use isDisabled and disabled at same time, how we know which is for styling which is prop of the component?